staging/kube-prometheus-stack/templates/ingress-rbac/grafana.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   name: dkp-{{ template "kube-prometheus-stack.fullname" . }}-grafana-admin
   labels:
     app: {{ template "kube-prometheus-stack.name" . }}-grafana
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
```

Comment:
since this chart is a fork, we have to apply changes as patches so we can easily sync with the upstream chart 

i.e. https://github.com/mesosphere/charts/tree/master/staging/kube-prometheus-stack/patch

you might be able to add to the existing patches though: https://github.com/mesosphere/charts/blob/master/staging/kube-prometheus-stack/patch/mesosphere/templates/ingress-rbac/grafana.yaml

Reply:
done!